SS_Activation_OpenMC.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Feb  8 08:16:12 2024
+
+@author: Anupama Rajendra
+"""
+import openmc
+import openmc.deplete
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+
+# Importing Vitamin-J energy group structure:
+# This excel file contains the energy bounds of the Vitamin J structure
+Vit_J = pd.read_excel('VitaminJEnergyGroupStructure.xlsx')
```

Comment:
It was having trouble reading from the txt file and giving a tally output. For some reason it appears to work fine with the Excel file

Reply:
I'm sure we can figure out how to get the txt file version to work - certainly better than adding proprietary software files.  It may be moot for this particular case given that we will likely switch to the built in Vit-J groups.